src/commands/es/insert.ts

```diff
@@ -0,0 +1,65 @@
+import { flags } from '@oclif/command'
+import { Kommand } from '../../common'
+import { Client } from '@elastic/elasticsearch'
+
+export default class EsInsert extends Kommand {
+  static description = 'Insert a document directly into ES (will replace if exists)'
+
+  static flags = {
```

Comment:
Since this is a TS file, this should expose an interface for flag named `IEsInsertFlag` ?

Reply:
Ditto

Reply:
Ditto